### PR TITLE
Fix missing jar and args passing for Pod deployment

### PR DESCRIPTION
**Pull request by misohu**
Resolving this issue https://github.com/canonical/kserve-rocks/issues/6 by wrapping the rock into docker image with correct ENTRYPOINT. Moreover I added also missing jar library which was not coppied from build stage. To properly build you need to:

```
rockcraft clean && rockcraft pack --verbosity=trace
sudo skopeo --insecure-policy copy oci-archive:pmml-server_v0.10.0_22.04_1_amd64.rock docker-daemon:base-pmmlserver:v0.10.0_22.04_1
docker build -t charmedkubeflow/pmmlserver:v0.10.0_22.04_1 . 
docker run charmedkubeflow/pmmlserver:v0.10.0_22.04_1
```

### CODE REVIEW

**Comment by i-chvets** [comment]
LGTM. Approved.
NOTE: Having ROCK wrapped in Docker defeats the purpose of having ROCKs. It makes the whole idea of using ROCKs in their current feature state very flawed. It would make sense to wait for Rockcraft to include features then to build ROCKs for sake of building ROCKs.

**Review by i-chvets — APPROVED** [review]
Approved.

File: pmmlserver/rockcraft.yaml [hunk 1]
```diff
@@ -34,35 +39,29 @@ parts:
     source-subdir: python
     source-tag: v0.10.0
     build-packages:
-    - wget
+    - python3.9
+    - python3-pip
+    - python3.9-distutils
     overlay-packages:
     - openjdk-11-jdk
+    - python3.9
```
Comment (SamirPS) [inline pmmlserver/rockcraft.yaml:47]: do you really need all this overlay package ? some can be on stage-packages no ?

**Review by SamirPS — CHANGES_REQUESTED** [review]

**Review by SamirPS — APPROVED** [review]

**Comment by SamirPS** [comment]
> LGTM. Approved. NOTE: Having ROCK wrapped in Docker defeats the purpose of having ROCKs. It makes the whole idea of using ROCKs in their current feature state very flawed. It would make sense to wait for Rockcraft to include features that support our use cases then to build ROCKs for sake of building ROCKs.

i agree with you

**Pull request merged**